client/shipping-label/state/actions.js

```diff
@@ -236,6 +237,13 @@ export const confirmAddressSuggestion = ( group ) => ( dispatch, getState, { sto
 		expandFirstErroneousStep( dispatch, getState, storeOptions, group );
 	};
 
+	const errors = getFormErrors( getState(), storeOptions );
+
+	// Make sure packaging is error free before fetching new rates after address confirmation
+	if ( hasNonEmptyLeaves( errors.packages ) ) {
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-services/pull/881/commits/3fd8ab429cc76e0b0fa3b89c112676cff6a1edf1.